org-code-javabuilder/lib/src/main/java/dev/javabuilder/LocalContentManager.java

```diff
@@ -28,9 +28,8 @@ public LocalContentManager() throws InternalServerError {
     this.projectData = projectData;
   }
 
-  @Override
-  public UserProjectFiles loadFiles() throws InternalServerError, UserInitiatedException {
```

Comment:
I think this change makes sense, but we'll need to pay attention to merge order because [this PR](https://github.com/code-dot-org/javabuilder/pull/246) conflicts with it.

Reply:
Sounds good, I'll go ahead and merge this now then